### PR TITLE
SRVOCF-289: Add the docs on invoking a function using 'kn func emit'

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+

https://issues.redhat.com/browse/SRVOCF-289

Docs preview: https://deploy-preview-34410--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-getting-started?utm_source=github&utm_campaign=bot_dp#serverless-invoke-func-kn_serverless-functions-getting-started

### CODE REVIEW

**Comment by rh-max** [comment]
@lance Could you please provide a review? https://github.com/openshift/openshift-docs/pull/34410#issue-685818708

**Comment by abrennan89** [comment]
@maximsvistunov I think this is overlapping a bit with the work I'm doing for `kn func emit` under https://issues.redhat.com/browse/SRVOCF-309 ? I'm not sure we need to add it to both places like this, rather than just giving a one line / step verification example after deploying a function that maybe says "you can invoke it using kn func emit"?

**Comment by lance** [comment]
> @maximsvistunov I think this is overlapping a bit with the work I'm doing for `kn func emit` under https://issues.redhat.com/browse/SRVOCF-309 ? I'm not sure we need to add it to both places like this, rather than just giving a one line / step verification example after deploying a function that maybe says "you can invoke it using kn func emit"?

Agreed this is probably redundant.

**Comment by rh-max** [comment]
@abrennan89 Missed it! Given that your emitting docs begin on literally the next line, how about adding just this simple line at the end of "Building and deploying..."?
https://github.com/openshift/openshift-docs/pull/34410/commits/5baf822f23584010bb527438e4442a1599d8f127
https://deploy-preview-34410--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-getting-started?utm_source=github&utm_campaign=bot_dp#serverless-functions-using-integrated-registry_serverless-functions-getting-started

**Comment by abrennan89** [comment]
@rh-max I'd say "You can verify..." but otherwise lgtm

**Comment by jrangelramos** [comment]
looks good to me

QE Approved

File: modules/serverless-functions-using-integrated-registry.adoc [hunk 1]
```diff
@@ -22,3 +22,5 @@ $ kn func build -r $(oc get route -n openshift-image-registry)
 ----
 $ kn func deploy -r $(oc get route -n openshift-image-registry)
 ----
++
+You can verify that the function deployed successfully by emitting a test event to it.
```
Comment (mburke5678) [inline modules/serverless-functions-using-integrated-registry.adoc:26]: My ignorance: Do users know how to emit a test? Do we need a link anywhere to show how to emit?

**Comment by mburke5678** [comment]
@rh-max One question you likely already considered yourself. Otherwise LGTM

File: modules/serverless-functions-using-integrated-registry.adoc (hunk 1 above)
Comment (rh-max) [reply modules/serverless-functions-using-integrated-registry.adoc:26]: @mburke5678 Good catch. What happened is the just-merged docs on emitting a test now begin literally on the next line after the line I've added:
https://github.com/openshift/openshift-docs/pull/33985
So we're good here.

**Review by lance — APPROVED** [review]

**Comment by abrennan89** [comment]
@rh-max @lance this is labelled as 4.8+ but I think that's incorrect, shouldn't it be 4.6+ if it's for 1.16.0?

**Comment by rh-max** [comment]
@abrennan89 @lance @jrangelramos @mburke5678 Thank yous for reviews! Ashleigh, could you please merge?

**Review by abrennan89 — APPROVED** [review]

**Pull request merged**